src/styles/_catalog.scss

```diff
@@ -1,22 +1,65 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$transition-duration: 300ms;
 
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  grid-template-rows: repeat(auto-fill, minmax(200px, 4fr));
+  gap: 48px;
   max-width: 944px;
+  justify-content: center;
   margin: 0 auto;
 
+  @media (min-width: 487px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(1, minmax(200px, 3fr));
+      grid-template-rows: repeat(auto-fill, minmax(200px, auto));
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(2, minmax(200px, 3fr));
+      grid-template-rows: repeat(auto-fill, minmax(200px, auto));
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      display: grid;
+      grid-template-columns: repeat(3, minmax(200px, 3fr));
+      grid-template-rows: repeat(auto-fill, minmax(200px, auto));
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      display: grid;
```

Comment:
Don't need to set dispaly grid in media queries, it won't change from what you set in catalog class